Action/ConvertPaymentAction.php

```diff
@@ -41,6 +41,7 @@ public function execute($request)
             'email' => $order->getClientEmail(),
             'firstName' => isset($d['firstName']) ? $d['firstName'] : '',
             'lastName' => isset($d['lastName']) ? $d['lastName'] : '',
+            'language' => $order->getLocale() ? substr($order->getLocale(), 0, 2) : '',
```

Comment:
I am not sure if $order always provides a getLocale method? Also we need to check if the result follows http://developers.payu.com/en/restapi.html#references_api_parameters - point 9.7